src/tests/packages/utils.unmocked.test.ts

```diff
@@ -0,0 +1,63 @@
+import { decodePngMetadata, decodeSvgMetadata } from "../../data/image";
```

Comment:
lets rename this file to `utils.test.ts` since it has the tests for the same file

Reply:
can't, already exists. I have spent some time yesterday figuring out how to have a test file that both tests with mocked and unmocked modules, and didn't figure it out.

Reply:
In that case, lets add a comment on top of this file